static/testData/test.dat

```diff
@@ -0,0 +1,32 @@
+1
+22
+0
+100
+200
+1000
+Temp
```

Comment:
I don't see this file being used anywhere, so you may consider removing it from the PR.